Makefile.am

```diff
@@ -740,10 +741,13 @@ EXTRA_DIST += \
 	man/glib-event-glue.c
 
 # ------------------------------------------------------------------------------
+rootlibexec_LTLIBRARIES += \
+	libshared.la
```

Comment:
shouldn't this library name include the systemd version string? otherwise it might happen that during RPM/dpkg updates processes are started that load the wrong version of this library, which will break given that we won't maintain compat on this.